problems/VDPTag/Project.toml

```diff
@@ -0,0 +1,27 @@
+name = "VDPTag2"
```

Comment:
I think you don't want the same Project.toml from the original project. Rather a new Project.toml with a different uuid.

Reply:
The easiest way to generate the Project.toml and the Manifest file is to do as follows: 
- go to `POMDPGallery/problems/YourPOMDP`
- create a `Project.toml` file (empty)
- run `julia --project` 
- add all the packages needed to run `script.jl` using the package manager (all the packages that are imported here for example: https://github.com/JuliaPOMDP/POMDPGallery.jl/blob/master/problems/RockSample/script.jl) 
- Check that `script.jl` runs fine in that environment 
- the Manifest.toml file should be created automatically.